sgkit-gpu-malariagen.ipynb

```diff
@@ -0,0 +1,284 @@
+{
```

Comment:
I would suggest moving this to a personal repo; I currently use https://github.com/eric-czech/sgkit-dev.  That's where I've been putting benchmark notebooks like this and it's a better place for the `dask-report*` files too.